kubernetes/05-secrets.yaml

```
@@ -5,5 +5,5 @@ metadata:
   namespace: easyshop
 type: Opaque
 stringData:
-  JWT_SECRET: "change-this-in-production"
-  NEXTAUTH_SECRET: "change-this-in-production"
\ No newline at end of file
+  JWT_SECRET: "cbmV4dC1hdXRoLXNlY3JldAo="
+  NEXTAUTH_SECRET: "and0LXNlY3JldAo="
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)